### PR TITLE
[#15] [UI] As a user, I can see the Survey Detail screen

### PR DESCRIPTION
**Pull request by luongvo**
https://github.com/luongvo/flutter-survey/issues/15

## What happened 👀

The user can start the survey by clicking on the arrow button on each survey in the survey list to open the Survey Detail screen.
 
## Insight 📝

- Click on the arrow button on each survey in the survey list to open the Survey Detail screen.
- On Survey Detail screen:
  - Show the back button
  - Show the title and description
  - Show the `Start Survey` button

### CODE REVIEW

File: lib/pages/surveydetail/widget/start_survey.dart [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      ],
+    );
+  }
+
+  Column _buildContent(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      children: [
+        Text(
+          // TODO
```
Comment (markgravity) [inline lib/pages/surveydetail/widget/start_survey.dart:67]: Can you describe more about this `TODO`? 🙏

File: lib/pages/surveydetail/widget/start_survey.dart [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+          "Working from home Check-In",
+          style: Theme.of(context).textTheme.headline5,
+        ),
+        const SizedBox(height: 10.0),
+        Text(
+          // TODO
```
Comment (markgravity) [inline lib/pages/surveydetail/widget/start_survey.dart:73]: Same here 🙏

File: lib/pages/surveydetail/widget/start_survey.dart [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+          ),
+        ),
+        Expanded(
+          child: Padding(
+            padding: const EdgeInsets.all(Dimens.defaultMarginPadding),
+            child: _buildContent(context),
+          ),
+        ),
+      ],
+    );
```
[the remark this answers is not in the document]
Comment (luongvo) [reply lib/pages/surveydetail/widget/start_survey.dart:59]: @markgravity I'll handle creating reusable widgets in following PRs 🏃

**Review by markgravity — APPROVED** [review]
Nice!! 🎉

**Pull request merged**